recipes/tmpdir-to-tmpdir/src/workflow.ts

```diff
@@ -26,6 +26,7 @@ export default function transform(root: SgRoot): string | null {
   let hasChanges = false;
 
   // Step 1: Find and update destructuring assignments from require('os') or require('node:os')
+	// @ts-ignore - ast-grep types are not fully compatible with JSSG types
```

Comment:
I think better to use `@ts-expect-error` so it alerts us when the issue is fixed.